backend/api/swagger/experiment.swagger.json

```diff
@@ -289,11 +280,20 @@
           "description": "Optional input field. Specify which resource this run belongs to.\nFor Experiment, the only valid resource reference is a single Namespace."
         },
         "storage_state": {
-          "$ref": "#/definitions/ExperimentStorageState",
+          "$ref": "#/definitions/apiExperimentStorageState",
           "description": "Output. Specifies whether this experiment is in archived or available state."
         }
       }
     },
+    "apiExperimentStorageState": {
```

Comment:
Where did this change come from?
It seems like a breaking change for other old clients.

Reply:
No :( I find it really hard to figure out where small changes like this actually comes from atm where several dependencies are update.

Reply:
I can help you take a look too when I have time

Reply:
was it correct before that we expect it to be `ExperimentStorageState`, https://github.com/kubeflow/pipelines/blob/a2c8042146724d883535c2ac5439c68c1fd86914/backend/api/swagger/experiment.swagger.json#L255 since it seems like they are supposed to allways have api in front?

Reply:
This is confusing.. I think it's due to latest generator. However, this is not a breaking change because request param has not changed. 

/cc @Bobgy @NikeNano

Reply:
Our internal build detect this change as well. Even proto file is not changed, swagger has been updated and forntend needs to generate corresponding apis to make build pass.

Reply:
> This is confusing.. I think it's due to latest generator. However, this is not a breaking change because request param has not changed.
> 

Yes, we should have investigate it further when we did the work. Good point @Jeffwan.